packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -257,22 +245,18 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         // instant relayer so that the l1Recipient cannot receive double payments. This means that once a relay is
         // disputed, it cant be sped up a second time (must finalize via the slow relay).
         uint256 priceRequestTime = getCurrentTime();
-        RelayData memory relayData =
-            RelayData({
-                relayId: numberOfRelays,
-                relayState: RelayState.Pending,
-                priceRequestTime: priceRequestTime,
-                realizedLpFeePct: realizedLpFeePct,
-                slowRelayer: msg.sender,
-                instantRelayer: relays[depositHash].instantRelayer
-            });
-        relays[depositHash] = relayData;
+        RelayData storage relayData = relays[depositHash];
```

Comment:
This syntax produces gas savings..?

Reply:
Kind of. I think it produces gas savings if you're not overwriting all fields (in this case we aren't overwriting the instant relayer. So, it doesn't need to read/write that field at all, which should save at least a few hundred gas.

When I tested it seemed to save a few k. I can get exact numbers if you'd like!

Reply:
That makes sense no need to get exact numbers if thats painful although that'd be insightful.

Reply:
I think it saves a SLOAD (aka a read).

Before:
```
Relay: 545882
```

After:
```
Relay: 543100
```